deploy/examples/cluster-cert-manager.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  name: cluster
+spec:
+  managementState: "Managed"
+  operatorLogLevel: "Debug"
```

Comment:
you probably don't want to deploy it like so by default

Reply:
Ok, removed.